app/src/main/java/com/sun/unsplash_01/ui/detail/PhotoDetailFragment.kt

```diff
@@ -0,0 +1,136 @@
+package com.sun.unsplash_01.ui.detail
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import com.sun.unsplash_01.R
+import com.sun.unsplash_01.data.model.PhotoDetail
+import com.sun.unsplash_01.data.source.local.sqlite.entity.ImageLocal
+import com.sun.unsplash_01.databinding.FragmentDetailBinding
+import com.sun.unsplash_01.extensions.hideStatusBar
+import com.sun.unsplash_01.extensions.toGone
+import com.sun.unsplash_01.extensions.toVisible
+import com.sun.unsplash_01.utils.Status
+import org.koin.android.ext.android.inject
+
+class PhotoDetailFragment : Fragment() {
+
+    private var isFavorite = false
+    private var photoID: String? = null
+    private var imageFavorite: ImageLocal? = null
+    private lateinit var binding: FragmentDetailBinding
+    private val photoDetailViewModel: PhotoDetailViewModel by inject()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            photoID = it.getString(BUNDLE_PHOTO_ID)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        requireActivity().hideStatusBar()
+        binding = DataBindingUtil.inflate<FragmentDetailBinding>(
+            inflater,
+            R.layout.fragment_detail,
+            container,
+            false
+        ).apply {
+            lifecycleOwner = this@PhotoDetailFragment
```

Comment:
```suggestion
            lifecycleOwner = this@PhotoDetailFragment.viewLifeCyclerOwner
```

Reply:
Cái này của thằng binding nên em dùng apply sau khi inflate xong thì em set lifeCycleOwner cho nó, set ở ngoài là phải gọi binding chấm đến ạ

Reply:
ý anh là truyền viewLifeCyclerOwner ý ? 
https://youtu.be/pErTyQpA390?t=307
https://github.com/android/architecture-samples/issues/675